ext/mbstring/mbstring.c

```diff
@@ -475,8 +482,13 @@ php_mb_parse_encoding_array(HashTable *target_hash, const mbfl_encoding ***retur
 				*entry++ = encoding;
 				n++;
 			} else {
-				php_error_docref(NULL, E_WARNING,
-					"Unknown encoding \"%s\"", ZSTR_VAL(encoding_str));
+				/* Called from an INI setting modification */
+				if (arg_num == 0) {
```

Comment:
This function cannot be used for ini settings (only strings make sense there), so this shouldn't be needed.

Reply:
Ah indeed will fix and commit